api/internal/managers/kubernetes/infra/install.go

```diff
@@ -144,7 +148,11 @@ func (m *infraManager) installAddOns(ctx context.Context, kcli client.Client, mc
 	return nil
 }
 
-func (m *infraManager) getAddonInstallOpts(license *kotsv1beta1.License, ki kubernetesinstallation.Installation) addons.KubernetesInstallOptions {
+func (m *infraManager) getAddonInstallOpts(ctx context.Context, license *kotsv1beta1.License, ki kubernetesinstallation.Installation) (addons.KubernetesInstallOptions, error) {
+	kotsadmNamespace, err := runtimeconfig.KotsadmNamespace(ctx, m.kcli)
```

Comment:
we shouldn't be using runtimeconfig package for kubernetes target installs. it does seem that the `KotsadmNamespace` isn't linux target specific though, so i wonder if it should not live in the runtimeconfig package, but another package that can be used by both linux/kubernetes targets. but i know we're not focusing on kubernetes target installs atm, so maybe just add a comment for now?